.github/workflows/deploy_pr_on_request.yml

```diff
@@ -0,0 +1,20 @@
+name: Trigger Deploy on Command
+
+on:
+  issue_comment:
+    types: [created]
+
+jobs:
+  deploy-test:
```

Comment:
Suggest we change that to `deploy-pr` which differentiates this from the job in `deploy.yml`.

Reply:
Good point - done.